jobs/generation/MetaGenerator.groovy

```diff
@@ -227,11 +227,13 @@ repos.each { repoInfo ->
         return;
     }
 
-    // Make the folders
+    // Make the folders. Save the root folder for overview generation
     def generatorFolder = ''
+    def projectFolder = ''
     for (folderElement in repoInfo.folders) {
         if (generatorFolder == '') {
             generatorFolder = folderElement
+            projectFolder = folderElement
```

Comment:
Where do you want this overview generation to go?  This will not go under dotnet_corefx/master, for instance.  I think you want to put it alongside in generatorFolder.